src/main/java/entities/HumanPlayer.java

```diff
@@ -0,0 +1,56 @@
+package entities;
+
+
+import java.util.*;
+
+public class HumanPlayer extends Player {
+
+    String name;
```

Comment:
Add access modifer

Reply:
Fixed this by adding name as an attribute to Player instead of only HumanPlayer.